src/main/resources/messages_de.properties

```diff
@@ -20,4 +20,6 @@ bookings.priority.HIGHEST=Hoch
 bookings.share.YES=Ja
 bookings.share.NO=Nein
 bookings.share.ON_REQUEST=Auf Anfrage
-bookings.share.tooltip=Dürfen andere den Raum zur gleichen Zeit nutzen?
\ No newline at end of file
+bookings.share.tooltip=Dürfen andere den Raum zur gleichen Zeit nutzen?
```

Comment:
According to GitHub, there may be an encoding issue with this file.
In a previous PR, I have changed this file to be UTF-8. Is that still the case? (IDEA should show the encoding at the bottom left)